crates/web5_cli/src/dids/create.rs

```diff
@@ -17,7 +17,12 @@ pub enum Commands {
         json_escape: bool,
     },
     Web {
+        #[arg(long)]
```

Comment:
```suggestion
```

The domain is a required parameter, so not an `arg` (you can just delete the line). 

Also, thoughts on calling it URL because it could include the HTTP path such as in [Example 5 here](https://w3c-ccg.github.io/did-method-web/#example-creating-the-did-with-optional-path). I'm thinking, name it `url`, and then handle the following use cases:

1. If they pass in `http://some-domain.com` then the `http` would throw an error, b/c needs to be `https`
2. With the caveat, `http` is allowed if the domain name is `localhost` (or `127.0.0.1`)
3. If they pass in `https://some-domain.com` then that's fine, but we have to splice off the `https://`
4. If they pass in the `did.json` file name (as the trailing path part), that's fine but it must be spliced off before creating the DID URI
5. If they pass in the `.well-known` use case in the `url` then it's also spliced off

Basically my thinking is, enable the developer to copy/paste exactly the endpoint they're expecting to host the DID Document at but then ensure the DID URI is spec conformant.

Reply:
Added a commit to cover these cases